transform/src/lib.rs

```diff
@@ -78,35 +67,234 @@ pub async fn initialize_staged(
     // - No runtime table operations
     // - No reference type globals
     // - Each module instantiated at most once
+    //
+    // instrumentation keeps track of all of the state which will be gathered from the instrumented
```

Comment:
```suggestion
    // `instrumentation` keeps track of all of the state which will be gathered from the instrumented
```
Just to make sure it's clear we're referring to a variable name here.